Add support for maximum replicas per node without stack (#1612)
docker/cli: docs/reference/commandline/service_create.md

`--placement-pref-rm` removes an existing placement preference that matches the
argument.

### Specify maximum replicas per node (--replicas-max-per-node)

> Use the `--replicas-max-per-node` flag to set the maximum number of replica task which can run on one Docker node.

Reviewer: Perhaps

```suggestion
Use the `--replicas-max-per-node` flag to set the maximum number of replica tasks that can run on a node.
```

The following command creates a nginx service with 2 replica tasks but only one replica task per node.

> One example of where this can be useful is to balance tasks over a set of datacenters together with `--placement-pref`

Reviewer: ```suggestion
One example where this can be useful is to balance tasks over a set of data centers together with `--placement-pref`
```
